student.rb

```diff
@@ -1,16 +1,19 @@
-class Student
+require_relative 'People'
+
+class Student < People
+
   attr_reader :age, :phase
   attr_accessor :name
```

Comment:
Hmm, you actually have them there already, plus writers for age and phase.  Remove these lines here, but also don't add writers where there weren't any before unless you actually need them (shouldn't need them here).